test/runtests.jl

```diff
@@ -2,7 +2,6 @@ using ImageEdgeDetection
 using Test, TestImages
 using ImageDraw
 using FileIO
-using ImageIO
```

Comment:
* ImageIO as a backend doesn't need to be manually imported; `FileIO.load` would automatically use that for PNG files. [FileIO registry](https://github.com/JuliaIO/FileIO.jl/blob/882dc754b9dce3b32def85a42717eefefb0fd1b6/src/registry.jl#L90-L98)
* ImageIO requires Julia >= 1.3
* relatively speaking, it would be more stable in test stage if we use the same IO backend.